ExtLibs/Maps/GMapMarkerBoat.cs

```diff
@@ -69,9 +69,26 @@ public override void OnRender(IGraphics g)
             {
             }
 
+            // Draw the boat with transparency (if possible)
+#if NET472_OR_GREATER
+            var colorMatrix = new System.Drawing.Imaging.ColorMatrix();
+            if(!IsActive)
+            {
+                colorMatrix.Matrix33 = 0.39f;

```

Comment:
no default app resources color ?

Reply:
All the GMapMarker implementations are a bit of a hardcoded nightmare. Handling this transparency value in a resource would be the right way to do it, but nothing else in them is being handled this way yet. They need a heavy refactoring.